server/api/person/person.subscribe.js

```diff
@@ -0,0 +1,27 @@
+const { emailPerson } = require('./person.email')
+const { config } = require('../../../config/clientConfig')
+const PubSub = require('pubsub-js')
+const { TOPIC_PERSON__CREATE, TOPIC_PERSON__EMAIL_SENT } = require('../../services/pubsub/topic.constants')
+
+/**
+ * This will be easier to add more status without having too much if. All we need is add another folder in email template folder and the status will reference to that folder
+ * @param {string} template status will be used to indicate which email template to use
+ * @param {object} to person email is for. (requestor or volunteer) with email populated.
+ * @param {object} props extra properties such as attachment
+ */
+const welcomeFrom = {
+  nickname: 'Welcome',
+  name: 'The team at Voluntarily',
+  email: 'welcome@voluntarily.nz'
+}
+
+module.exports = (server) => {
```

Comment:
I see `server` isn't used here - I'm assuming it's here because it's essentially an interface and `server` may be used in other `*.subscribe.js` functions?

Reply:
thats correct